src/main/resources/checkstyle/checkstyle-suppressions-legacy.xml

```diff
@@ -44,7 +44,7 @@
   <suppress checks="UnnecessaryParentheses" files=".*[/\\]SparseGradient\.java$" lines="385,396" />
   <suppress checks="UnnecessaryParentheses" files=".*[/\\]DerivativeStructure\.java$" lines="468,481" />
   <suppress checks="UnnecessaryParentheses" files=".*[/\\]SimpleCurveFitter\.java$" lines="321-322" />
-  <suppress checks="UnnecessaryParentheses" files=".*[/\\]LevenbergMarquardtOptimizer\.java$" lines="527,741" />
+  <suppress checks="UnnecessaryParentheses" files=".*[/\\]LevenbergMarquardtOptimizer\.java$" lines="525,527,741" />
```

Comment:
Since you removed two lines above `527` this should be `"525,739"`. Please change and verify with `mvn checkstyle:check` in this module.

Reply:
done.